### PR TITLE
Remove useless duplicated code

### PR DESCRIPTION
**Pull request by tifabien**
I am targeting this branch, because it removes useless duplicated code. The code is already the same in the `FileProvider::getReferenceImage` parent method

### CODE REVIEW

File: src/Provider/ImageProvider.php [hunk 1]
```diff
@@ -127,17 +127,6 @@ public function getHelperProperties(MediaInterface $media, $format, $options = [
         return array_merge($params, $options);
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    public function getReferenceImage(MediaInterface $media)
```
Comment (kunicmarko20) [inline src/Provider/ImageProvider.php]: Method is public and removing it would be a BC Break, you can deprecate it if you want.

Comment (greg0ire) [reply src/Provider/ImageProvider.php]: Why would it be a BC break if the parent method exists?

Comment (kunicmarko20) [reply src/Provider/ImageProvider.php]: My bad, didn’t understand

**Comment by tifabien** [comment]
I can see that indeed the lint build failed but I don't know why... Is there a way to rebuild the task ?

**Comment by kunicmarko20** [comment]
restarted

**Comment by kunicmarko20** [comment]
@greg0ire do we need a changelog here?

**Comment by OskarStark** [comment]
> @greg0ire do we need a changelog here?

lets keep the changelog and tag it as minor

**Comment by greg0ire** [comment]
I would remove the changelog and keep it pedantic: imagine you update the lib, you will not notice any changes, will you? I mean the public API is the same, and the behavior is the same.

**Comment by tifabien** [comment]
I agree to not include the changelog as the public API is the same. That's invisible to the user.

**Comment by greg0ire** [comment]
It's settled then, I removed the changelog

**Comment by greg0ire** [comment]
Thanks @tifabien !

**Pull request merged**